src/material-experimental/mdc-list/list-base.ts

```diff
@@ -28,20 +34,12 @@ function toggleClass(el: Element, className: string, on: boolean) {
   }
 }
 
-@Directive()
-/** @docs-private */
-export abstract class MatListBase {
-  // @HostBinding is used in the class as it is expected to be extended. Since @Component decorator
-  // metadata is not inherited by child classes, instead the host binding data is defined in a way
-  // that can be inherited.
-  // tslint:disable-next-line:no-host-decorator-in-concrete
-  @HostBinding('class.mdc-list--non-interactive')
-  _isNonInteractive: boolean = false;
-}
-
 @Directive()
 /** @docs-private */
 export abstract class MatListItemBase implements AfterContentInit, OnDestroy, RippleTarget {
+  @HostBinding('tabindex')
+  _tabIndex = -1;
```

Comment:
Why does `tabindex = -1` go in the base class?

Reply:
This was meant to mimic the logic in MDC's component that initializes list items to `tabIndex=-1`. However, putting it here would affect regular `mat-list` as well, so I moved the logic to `MatInteractiveListBase` instead